dotnet/notebooks/10-BingSearch-using-kernel.ipynb

```diff
@@ -42,7 +42,7 @@
     "#!import config/Utils.cs\n",
     "\n",
     "using Microsoft.SemanticKernel;\n",
-    "using Microsoft.SemanticKernel.Skills.Core;\n",
+    "Microsoft.SemanticKernel.Plugins.Core;\n",
```

Comment:
missing `using`

```suggestion
    "using Microsoft.SemanticKernel.Plugins.Core;\n",
```

Reply:
Removing notebook changes, they shouldn't be part of this PR